### PR TITLE
News post & EB Resolutions for 9 Dec

### PR DESCRIPTION
**Pull request by emmahodcroft**
Adds the news post and associated EB resolutions for 9 Dec

[Preview](https://preview-news-post-4.pathoplexus.org/)

Please check:
- [News index page](https://preview-news-post-4.pathoplexus.org/news/)
- [New news page itself](https://preview-news-post-4.pathoplexus.org/news/2024-12-09-expanding-pathoplexus-mpox)
  - All links within --- note that the link to the EB resolutions won't work as it's part of this PR 🙃 
- [Minutes & resolutions index page](https://preview-news-post-4.pathoplexus.org/about/governance/minutes)
- [2024-12-09 EB resolutions PDF](https://preview-news-post-4.pathoplexus.org/about/governance/minutes/2024-12-09_EB_Resolutions.pdf)

### CODE REVIEW

File: monorepo/website/src/pages/news/2024-12-09-adding-mpox.mdx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+layout: ../../layouts/MdLayout.astro
+title: "Addition of mpox"
+---
+
+<div>
+    <a href="/news">News</a> > <b>Addition of mpox</b>
+</div>
+
+# Addition of mpox
```
Comment (corneliusroemer) [inline monorepo/website/src/pages/news/2024-12-09-adding-mpox.mdx:10]:
```suggestion
# Addition of mpox and other news
```

The post is about more than mpox ;)

**Review by corneliusroemer — COMMENTED** [review]
We should agree on one of a) Adding mpox or b) Addition of mpox

Brave Browser 2024-12-09 15 04 59

image

File: monorepo/website/src/pages/news/2024-12-09-adding-mpox.mdx (hunk 1 above)
Comment (emmahodcroft) [reply monorepo/website/src/pages/news/2024-12-09-adding-mpox.mdx:10]: I just didn't know if that's a very catchy title?

File: monorepo/website/src/pages/news/index.astro [hunk 2]
```diff
@@ -3,6 +3,13 @@ import MdLayout from '../../layouts/MdLayout.astro';
 
 // Add new news items here
 const newsItems = [
+    {
+        slug: '2024-12-09-adding-mpox',
+        title: 'Adding mpox',
+        date: '9 December 2024',
+        excerpt:
+            "Today Pathoplexus launches its first additional viral pathogen: mpox, or MPXV (previously known as monkeypox). Within the context of the current mpox outbreaks and ongoing health concerns, Pathoplexus aims to support sequence sharing to improve both understanding and response…",
```
Comment (theosanderson) [inline monorepo/website/src/pages/news/index.astro:11]:
```suggestion
            "Today Pathoplexus launches its first additional viral pathogen: mpox, or MPXV (previously known as monkeypox). In the context of the current mpox outbreaks and ongoing health concerns, Pathoplexus aims to support sequence sharing to improve both understanding and response…",
```
For readability - feel free to reject or apply :)

File: monorepo/website/src/pages/news/index.astro [hunk 3]
```diff
@@ -3,6 +3,13 @@ import MdLayout from '../../layouts/MdLayout.astro';
 
 // Add new news items here
 const newsItems = [
+    {
+        slug: '2024-12-09-adding-mpox',
```
Comment (corneliusroemer) [inline monorepo/website/src/pages/news/index.astro:7]:
```suggestion
        slug: '2024-12-09-addition-of-mpox',
```

File: monorepo/website/src/pages/news/index.astro [hunk 4]
```diff
@@ -3,6 +3,13 @@ import MdLayout from '../../layouts/MdLayout.astro';
...
+        title: 'Adding mpox',
```
Comment (corneliusroemer) [inline monorepo/website/src/pages/news/index.astro:8]:
```suggestion
        title: 'Addition of mpox',
```

File: monorepo/website/src/pages/news/2024-12-09-adding-mpox.mdx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+*By the Pathoplexus Team - 9 December 2024*
+
+Today Pathoplexus launches its first additional viral pathogen: mpox, or MPXV (previously known as monkeypox). Within the context of the current mpox outbreaks and ongoing health concerns, Pathoplexus aims to support sequence sharing to improve both understanding and response.
```
Comment (corneliusroemer) [inline monorepo/website/src/pages/news/2024-12-09-adding-mpox.mdx:14]:
```suggestion
Today Pathoplexus launches its first additional viral pathogen: mpox, or MPXV (previously known as monkeypox). In the context of the current mpox outbreaks and ongoing health concerns, Pathoplexus aims to support sequence sharing to improve both understanding and response. 
```

mirroring @theosanderson's suggested change

File: monorepo/website/src/pages/news/index.astro [hunk 6]
```diff
@@ -3,6 +3,13 @@ import MdLayout from '../../layouts/MdLayout.astro';
 
 // Add new news items here
 const newsItems = [
+    {
+        slug: '2024-12-09-expanding-pathoplexus',
```
Comment (corneliusroemer) [inline monorepo/website/src/pages/news/index.astro:7]:
```suggestion
        slug: '2024-12-09-expanding-pathoplexus-mpox',
```

Comment (corneliusroemer) [reply monorepo/website/src/pages/news/index.astro:7]: out of SEO superstition I'll add mpox, so it shows up higher up when people google mpox

**Review by corneliusroemer — APPROVED** [review]

**Pull request merged**